README.md

```diff
@@ -45,42 +45,20 @@ pip install git+https://github.com/GoogleCloudPlatform/cloud-sql-python-connecto
 
 ### How to use this connector
 
-To use the connector: import the connector by including the following statement at the top of your Python file:
+To use the connector: import the connector and SQLAlchemy by including the following statements at the top of your Python file:
 ```Python
 from google.cloud.sql.connector import connector
+import sqlalchemy
 ```
 
-Use the connector to create a connection object by calling the connect method. Input your connection string as the first positional argument and the name of the database driver for the second positional argument. Insert the rest of your connection keyword arguments like user, password and database. You can also set the optional `timeout` or `ip_type` keyword arguments.
+The connector itself creates connection objects by calling its `connect` method but does not manage database connection pooling. For this reason, it is recommended to use the connector alongside a library that can create connection pools, such as [SQLAlchemy](https://www.sqlalchemy.org/). This will allow for connections to remain open and be reused, reducing connection overhead and the number of connections needed.
 
-```
-conn = connector.connect(
-    "project:region:instance",
-    "pymysql",
-    user="root",
-    password="shhh",
-    db="your-db-name"
-... insert other kwargs ...
-)
-```
-
-The returned DB-API 2.0 compliant connection object can then be used to query and modify the database:
-```
-# Execute a query
-cursor = conn.cursor()
-cursor.execute("SELECT * from my_table")
-
-# Fetch the results
-result = cursor.fetchall()
-
-# Do something with the results
-for row in result:
-    print(row)
-```
+In the connector's `connect` method below, input your connection string as the first positional argument and the name of the database driver for the second positional argument. Insert the rest of your connection keyword arguments like user, password and database. You can also set the optional `timeout` or `ip_type` keyword arguments.
 
 To use this connector with SQLAlchemy, use the `creator` argument for `sqlalchemy.create_engine`:
 ```
-def getconn() -> pymysql.connections.Connection:
-    conn: pymysql.connections.Connection = connector.connect(
+def getconn():
```

Comment:
Why did we drop the typing?

Reply:
I can add it back in if you want? I just thought it made the sample look a little cluttered. I'm open to adding it back in.

Reply:
I like typing. It's usually valuable if you're using typing, and easy to ignore if you aren't.